polkadot/runtime/parachains/src/inclusion/mod.rs

```diff
@@ -264,9 +267,32 @@ impl From<u32> for AggregateMessageOrigin {
 	}
 }
 
+impl From<ParaId> for AggregateMessageOrigin {
```

Comment:
If we have this `impl` now, can we remove the one above (`impl From<u32> for AggregateMessageOrigin {`) ?